File: 2163/2163.cs
public class Solution {
    public int MinimumDifferenceInSumsAfterRe(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(MinimumDifferenceInSumsAfterRe(root.left), MinimumDifferenceInSumsAfterRe(root.right));
    }
}

File: 2163/2163.cpp
class Solution {
public:
    int minimumDifferenceInSumsAfterRe(TreeNode* root) {
        if(!root) return 0;
        return 1+max(minimumDifferenceInSumsAfterRe(root->left),minimumDifferenceInSumsAfterRe(root->right));
    }
};